src/modules/ethereum/chains/ChainConfigStore.ts

```diff
@@ -23,35 +39,40 @@ function remove<T>(arr: T[], predicate: (item: T) => boolean) {
 }
 
 export class ChainConfigStore extends PersistentStore<ChainConfig> {
```

Comment:
I think `ChainConfigStore` class isn't used anywhere outside? Let's remove this export